### ISSUE TITLE
[JENKINS-3891] Build 'appears to be stuck' even if in queue

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#3492]
<p>If I understand the abort stuck builds feature, it is intended to kill builds<br/>
that hang due to some sort of execution.  However, builds are reported to be<br/>
stuck even if they are simply waiting their turn in the queue.  It seems like<br/>
this timer should not start until the job begins execution.</p>

---
<details><summary><i>Originally reported by <a title="John McNair" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=pamdirac">pamdirac</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-3891" target="_blank">Build 'appears to be stuck' even if in queue</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Kohsuke Kawaguchi" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kohsuke">kohsuke</a>
<li><b>reported by</b>: pamdirac
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2009-06-23T15:36:37+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 0
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jglick">jglick</a>:</i>
<ul>
	<li>
	<ul>
		<li>
		<ul>
			<li>Issue 3040 has been marked as a duplicate of this issue. ***</li>
		</ul>
		</li>
	</ul>
	</li>
</ul>
